extensions/amp-carousel/0.1/carousel-controls.js

```diff
@@ -47,8 +48,8 @@ export class CarouselControls {
    * Meant to be called during carousel's buildCallback().
    */
   setupBehaviors_() {
-    this.setupButtonInteraction(this.prevButton_, () => this.handlePrev());
-    this.setupButtonInteraction(this.nextButton_, () => this.handleNext());
+    this.setupButtonInteraction(this.prevButton_, this.handlePrev.bind(this));
+    this.setupButtonInteraction(this.nextButton_, this.handleNext.bind(this));
```

Comment:
AFAICT these are the the same/this change is a no-op. Given that, would they generate different serialized output?

Reply:
Yep! One has a name, the other is an anonymous function. `bound handlePrev` vs. `""`